### PR TITLE
Add new fluent-bit filters/parsers

### PR DESCRIPTION
**Pull request by Kristian-ZH**
**How to categorize this PR?**

/area logging
/kind enhancement
/priority normal

**What this PR does / why we need it**:
Add new fluent-bit filters/parsers + some replacements for better readability

**Which issue(s) this PR fixes**:
https://github.com/gardener/gardener/issues/2677

**Special notes for your reviewer**:
@vlvasilev 
**Release note**:

```improvement operator
None
```

### CODE REVIEW

File: charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml [hunk 1]
```diff
@@ -482,7 +471,7 @@ data:
       elseif severity == "W" or severity == "WRN" or severity == "WARN" or severity == "WARNING" then
         record["severity"] = "WARN"
         modified = true
-      elseif severity == "E" or severity == "ERR" or severity == "ERROR" then
+      elseif severity == "E" or severity == "ERR" or severity == "ERROR" or severity == "eror" then
```
Comment (ialidzhikov) [inline charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml]: Why is that needed?

Comment (Kristian-ZH) [reply charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml]: grafana

Comment (Kristian-ZH) [reply charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml]: t=2020-08-17T09:40:08+0000 lvl=`eror` msg="..." logger=context error="..."

Comment (ialidzhikov) [reply charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml]: Several lines above there is `      severity = trim(severity):upper()`. I guess comparing to lowercase log level won't catch anything.

Comment (Kristian-ZH) [reply charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml]: yup, i will change it to `EROR`, thanks!

**Review by vlvasilev — DISMISSED** [review]
/lgtm

File: charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml [hunk 2]
```diff
@@ -171,23 +166,24 @@ data:
...
 
     [FILTER]
         Name                parser
-        Match               kubernetes.*cluster-autoscaler*cluster-autoscaler*
+        Match               kubernetes.*kube-state-metrics*kube-state-metrics*
         Key_Name            log
         Parser              kubeapiserverParser
         Reserve_Data        True
 
+    # Kube-system filters
```
Comment (vpnachev) [inline charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml]:
```suggestion
    # System components filters
```

File: charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml [hunk 3]
```diff
@@ -270,52 +252,67 @@ data:
...
 
     [FILTER]
-        Name record_modifier
-        Match *kube-controller-manager*
-        Record type user
+        Name modify
+        Match kubernetes.*gardener-resource-manager*gardener-resource-manager*
+        Rename level severity
+        Rename msg log
+        Rename logger source
```
Comment (vpnachev) [inline charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml]: Add some indentation between the key and the value.

File: charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml [hunk 4]
```diff
@@ -220,31 +209,24 @@ data:
 
     [FILTER]
         Name                parser
-        Match               kubernetes.*vpn-seed*vpn-seed*
```
Comment (vpnachev) [inline charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml:223]: Why is this removed?

File: charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml [hunk 5]
```diff
@@ -220,31 +209,24 @@ data:
...
-        Key_Name            log
-        Parser              vpnshootParser
-        Reserve_Data        True
-
-    [FILTER]
-        Name                parser
-        Match               kubernetes.*addons-kubernetes-dashboard*kubernetes-dashboard*
```
Comment (vpnachev) [inline charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml:230]: Why is this removed?

File: charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml [hunk 6]
```diff
@@ -220,31 +209,24 @@ data:
...
+        Match               kubernetes.*node-exporter*node-exporter*
         Key_Name            log
-        Parser              kubernetesdashboardParser
+        Parser              nodeexporterParser
         Reserve_Data        True
 
     [FILTER]
         Name                parser
-        Match               kubernetes.*coredns*coredns*
```
Comment (vpnachev) [inline charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml]: Why is this removed?

**Review by vpnachev — CHANGES_REQUESTED** [review]

**Review by rfranzke — DISMISSED** [review]

**Review by rfranzke — DISMISSED** [review]

**Review by rfranzke — DISMISSED** [review]

File: charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml [hunk 7]
```diff
@@ -136,14 +138,14 @@ data:
...
-        Match               kubernetes.*kube-controller-manager*kube-controller-manager*
+        Match               kubernetes.*kube-apiserver*vpn-seed*
         Key_Name            log
-        Parser              kubeapiserverParser
+        Parser              vpnshootParser
         Reserve_Data        True
 
     [FILTER]
         Name                parser
-        Match               kubernetes.*kube-scheduler*cloud-kube-scheduler*
```
Comment (vpnachev) [inline charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml:146]: Please, bring back this filter.

File: charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml [hunk 8]
```diff
@@ -270,52 +274,67 @@ data:
...
+        Key_Name            log
+        Parser              gardenerResourceManagerParser
+        Reserve_Data        True
 
     [FILTER]
-        Name record_modifier
-        Match *kube-controller-manager*
-        Record type user
+        Name modify
+        Match kubernetes.*gardener-resource-manager*gardener-resource-manager*
```
Comment (vpnachev) [inline charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml]: Add Indentation please.

File: charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml [hunk 9]
```diff
@@ -270,52 +274,67 @@ data:
...
 
     [FILTER]
-        Name record_modifier
-        Match *cloud-controller-manager*
-        Record type user
+        Name modify
+        Match kubernetes.*gardener-extension*
+        Rename level severity
+        Rename msg log
+        Rename logger source
```
Comment (vpnachev) [inline charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml]: Add Indentation please.

**Review by vpnachev — CHANGES_REQUESTED** [review]

File: charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml (hunk 7 above)
Comment (Kristian-ZH) [reply charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml:146]: This filter is there. Just was duplicated.

Comment (vpnachev) [reply charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml:146]: True. I had to open the full file, not just checking the diff provided here.

**Review by vpnachev — APPROVED** [review]
/lgtm

**Review by rfranzke — APPROVED** [review]

**Review by vlvasilev — APPROVED** [review]
/lgtm
